src/inc/msquic.h

```diff
@@ -846,6 +852,7 @@ typedef struct QUIC_SCHANNEL_CREDENTIAL_ATTRIBUTE_W {
 #endif
 #define QUIC_PARAM_CONN_STATISTICS_V2                   0x05000016  // QUIC_STATISTICS_V2
 #define QUIC_PARAM_CONN_STATISTICS_V2_PLAT              0x05000017  // QUIC_STATISTICS_V2
+#define QUIC_PARAM_CONN_RESUMPTION_TICKET_VALID         0x05000018  // uint8_t (BOOLEAN)
```

Comment:
I think we're actually going to want to make a new function in `QUIC_API_TABLE` for this. The problem with using a SetParam is that it is a blocking call to the app, which we don't want in this case. I think we want to create a `ConnectionResumptionTicketValidationComplete` function...